docs/docs/tutorials/creating-new-environments.md

```diff
@@ -1,181 +1,94 @@
-
-Some useful sections to notice in the `user` page are:
-
-- **User**: this section of the dashboard allows users to explicitly logout of the interface.
-- **Namespaces**: `Namespaces` are an important part of the `conda-store` [authorization model][conda-store-docs-auth]. They
-  control what level of access users are provided to the individual environments. In other words, based on your
-  permissions in each namespace, your ability to create, read, update, or delete and environment will differ.
-- **Permissions**: your current permissions in each namespace.
-
-That is, unless your admin has configured your namespace differently. You can see here that the `conda-store`
-authorization model is able to provide admins with a fine-grained level of control.
-
-## Step 2 - Creating a new environment
-
-1. To create a new environment, click on the `Create New Environment` button on the top right of the navigation bar. You will
-   be presented with an option to upload a
-   [conda YAML file](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-from-an-environment-yml-file)
-   or write your own.
+Do not install libraries directly in the notebook or through the terminal (outside a conda-store managed environment). It won't work as expected and may break your instance in subtle ways.
+:::
 
-   ![conda-store create environment UI](/img/tutorials/conda_store_create_env.png)
+## Preliminary reading
 
-   For this tutorial, you can copy and paste the following environment specification:
+It's useful to understand basics of conda-store and how it builds on top of the conda ecosystem, to use it effectively in Nebari.
 
-   ```yaml title="Sample environment specification"
-   channels:
-   - conda-forge
-   dependencies:
-   - python=3.9
-   - numpy
-   - matplotlib
-   - pandas
-   - panel
-   - ipykernel
-   - ipywidgets
-   name: example-environment
-   prefix: null
-   ```
+- [Essential conda concepts: Packages, environments, channels, dependencies, etc.](https://conda.store/conda-store/explanations/conda-concepts)
+- [conda-store concepts: Reproducibility, namespaces, environment versions, roles, etc.](https://conda.store/conda-store/explanations/conda-store-concepts)
 
-2. After you copy the above into the UI, go ahead and click the `Submit` button.
+Nebari has conda-store integrated, and you can use it through the graphical UI.
 
-3. You will be redirected to the environments overview page. Find the card with the name of the new environment under your user's namespace (e.g.
-   `<your-username>/example-environment`).
+## Open `conda-store` web interface
 
-   ![Newly create environment card](/img/tutorials/conda_store_new_env.png)
+To navigate to conda-store:
 
-4. Click on the card for your newly created environment. The UI will display the environment specification:
+* From Nebari Home, click on **"Environment Management"** under "Service"
```

Comment:
```suggestion
* From Nebari Home, click on **"Environment Management"** under "Services"
```
"Nebari Home" is kind of ambiguous. I'm assuming you mean the App Launcher? An image may help here. 

Also, the App Launcher is optional so should we point that out in this option